src/number_theory/number_field.lean

```diff
@@ -53,7 +53,9 @@ protected lemma is_algebraic : algebra.is_algebraic ℚ K := algebra.is_algebrai
 
 /-- The ring of integers (or number ring) corresponding to a number field
 is the integral closure of ℤ in the number field. -/
-@[nolint unused_arguments] -- There are multiple definitions of rings of integers.
+-- The `is_number_field K` hypothesis is not used but is required for
+-- this definition to make sense.
+@[nolint unused_arguments]
```

Comment:
Shouldn't we leave it out anyway? It would be sufficient to assume it for lemmas, right?

Reply:
I'm just a bit wary of someone having `open function_field number_field` somewhere, trying to get the ring of integers of a function field, and receiving the integral closure of `ℤ` instead. Complaining about a missing instance would prevent that issue. But I guess they will notice the issue as soon as they try to prove anything with it.